aspnetcore/migration/webapi.md

```diff
@@ -96,11 +101,19 @@ Configure routing as follows:
     The preceding [[Route]](xref:Microsoft.AspNetCore.Mvc.RouteAttribute) attribute configures the controller's attribute routing pattern. The [[ApiController]](xref:Microsoft.AspNetCore.Mvc.ApiControllerAttribute) attribute makes attribute routing a requirement for all actions in this controller.
 
     Attribute routing supports tokens, such as `[controller]` and `[action]`. At runtime, each token is replaced with the name of the controller or action, respectively, to which the attribute has been applied. The tokens reduce the number of magic strings in the project. The tokens also ensure routes remain synchronized with the corresponding controllers and actions when automatic rename refactorings are applied.
+1. Set the project's compatibility mode to ASP.NET Core 2.2:
+
+    [!code-csharp[](webapi/sample/ProductsCore/Startup.cs?name=snippet_ConfigureServices&highlight=4)]
+
+    The preceding change:
+
+    * Is required to use the `[ApiController]` attribute at the controller level.
+    * Opts in to potentially breaking behaviors introduced in ASP.NET Core 2.2.
```

Comment:
Is "Opts in" hyphenated? ... I would think so, but I might be stuck in AP styles again! lol

... or it's the **_cough syrup_** talking. 🚑

Reply:
My gut tells me no hyphen in this case.

Reply:
ur prob right. ..... but now, I wonder if it's "Opts into potentially ..." :smile: